docker-compose-production.yml

```diff
@@ -167,7 +167,7 @@ services:
       - prometheus_config
     command:
       - "--config.file=/run/secrets/prometheus_config"
-      - "--storage.tsdb.retention.time=30d"
+      - "--storage.tsdb.retention.time=90d"
```

Comment:
@jhavrlant Pokud je to takto snadné, navrhuji to upravit na 360d a případně to pak omezit, pokud by se to ukázalo jako z nějakého důvodu neudržitelné. Lze odhadnout, jak moc nám to naroste?

Reply:
Opraveno